GW2EIEvtcParser/LogLogic/Raids/W8/Bosses/DecimaTheStormsinger.cs

```diff
@@ -830,6 +829,35 @@ private static void AddThunderAoE(SingleActor actor, ParsedEvtcLog log, CombatRe
         }
     }
 
+    /// <summary>
+    /// Red death zone underneath Decima during Earthrend, Seismic Crash and Seismic Reposition.
+    /// </summary>
+    /// <param name="time">Time at which the other mechanic effect plays.</param>
+    /// <param name="isEarthrend">Wether the mechanic is Earthrend of a Seismic.</param>
+    private static void AddDeathZone(NPC target, ParsedEvtcLog log, CombatReplay replay, long time, bool isEarthrend)
+    {
+        (long start, long end) lifespan;
+
+        if (log.CombatData.TryGetEffectEventsBySrcWithGUID(target.AgentItem, EffectGUIDs.DecimaJumpAoEUnderneath, out var deathZone))
```

Comment:
This looks not very efficient, would be better to create a ComputeLifespanWithSecondaryEffectsAndPosition that takes an array of GUIDs that use TryGetEffectEventsBySrcWithGUIDs and have only a single loop instead of two